jsonb-generator/src/main/java/io/avaje/jsonb/generator/PropertyIgnoreReader.java

```diff
@@ -29,9 +30,14 @@ final class PropertyIgnoreReader {
     ignoreSerialize = propertyMethodOverride;
 
     final IgnorePrism ignored = IgnorePrism.getInstanceOn(element);
+
+    final IgnorePrism ignoreFromType = IgnorePrism.getInstanceOn(element);
```

Comment:
Huh? The `ignoreFromType` is the same as `ignored` (uses the same element) ... so it doesn't actually look at the type? 

When I add a blackbox test it also doesn't seem to work (but I don't see how it would work when ignoreFromType is just the same as ignored?